programmers/난이도별/level01.같은_숫자는_싫어/JiaLee0707.js

```diff
@@ -0,0 +1,3 @@
+function solution(arr) {
+    return arr.filter((r, index) =>  r !== arr[index + 1]);
```

Comment:
연속되는 숫자의 특성을 잘 사용해서 문제를 풀어주신 것 같아요!
조금 더 개선할 점이 있다면 **filter**내에서 `index`라는 값은 한 눈에 파악이 가능하지만 `r`이라는 값은 **단번에 파악하기 어렵다**고 느껴져요.

아래와 같이 **value**(_더 좋은 의견이 있다면 알려주세요!_)로 선언하는 게 더 보기 쉬울 것 같아 변수 명만 수정해보았습니다 👍 
```suggestion
    return arr.filter((value, index) =>  value !== arr[index + 1]);
```